beacon-chain/core/blocks/header_test.go

```diff
@@ -189,6 +189,10 @@ func TestProcessBlockHeader_PreviousBlockRootNotSignedRoot(t *testing.T) {
 }
 
 func TestProcessBlockHeader_SlashedProposer(t *testing.T) {
+	cfg := params.BeaconConfig().Copy()
+	cfg.MinGenesisActiveValidatorCount = 16384
+	params.OverrideBeaconConfig(cfg)
+
 	validators := make([]*ethpb.Validator, params.BeaconConfig().MinGenesisActiveValidatorCount)
```

Comment:
Why `MinGenesisActiveValidatorCount` changed?

Reply:
It was updated at #33. (16384 -> 8192)

Reply:
As you mentioned, it would be much better to change validator index instead of overriding config. I will handle it.